parsec/backend/postgresql/migrations/0001_initial.sql

```diff
@@ -1,6 +1,18 @@
 -- Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
 
 
+-------------------------------------------------------
+--  Migration
+-------------------------------------------------------
+
+
+CREATE TABLE migration (
+    _id SERIAL PRIMARY KEY,
```

Comment:
Why not use the id present in the migration name as the primary key ? (name field is still useful for readable informations)

It should simplify the `_idx_limit` function by removing the need for `_last_applied_migration_file_idx`

What should be done:
- add a comment here ;-)
- replace the SERIAL type by INTEGER
- explicitly provide the id field in the INSERT statement

Reply:
Done